pkg/operator/ceph/csi/spec.go

```diff
@@ -381,6 +386,7 @@ func (r *ReconcileCSI) startDrivers(ownerInfo *k8sutil.OwnerInfo) error {
 			applyLogrotateSidecar(&cephfsProvisionerDeployment.Spec.Template, "csi-cephfs-deployment-log-collector", LogrotateTemplatePath, tp)
 		}
 		cephfsProvisionerDeployment.Spec.Template.Spec.HostNetwork = opcontroller.EnforceHostNetwork()
+		cephfsProvisionerDeployment.Spec.RevisionHistoryLimit = CSIParam.RevisionHistoryLimit
```

Comment:
Do we need to use `CSIParam` here? Why use this method instead of using `opcontroller.RevisionHistoryLimit()` in the same way that we use `opcontroller.EnforceHostNetwork()` just as in the line above?

Reply:
Originally, I thought we need this CSIParam variable. But changed to `opcontroller.RevisionHistoryLimit()`now (in all places).